configure: error: empy not detected despite being installed

# empy is required
Although I have installed empy `pip install empy` and `pip3 install empy`.
When I tried running `autogen.sh`, empy is required error keeps on showing.
```bash
checking for empy... no
checking for empy3... no
configure: error: empy is required
```

--- a/configure.ac
+++ b/configure.ac
@@ -15,10 +15,7 @@ AM_MAINTAINER_MODE
 PYTHON=python3
 AM_PATH_PYTHON
 
-AC_PATH_PROGS([EMPY], [empy empy3])
-if test -z "$EMPY"; then
-	AC_MSG_ERROR([empy is required])
-fi
+AS_IF([$PYTHON -m em >/dev/null 2>&1], AC_MSG_RESULT([empy found]), AC_MSG_ERROR([empy not found]))
 
 PKG_CHECK_MODULES(SHELL, gtk+-3.0)
 

--- a/data/Makefile.am
+++ b/data/Makefile.am
@@ -1,12 +1,12 @@
 SUBDIRS = icons
 
 sugar-72.gtkrc: gtkrc.em
-	$(PYTHON) - -D scaling=\'72\' $(srcdir)/gtkrc.em > \
-		$(top_builddir)/data/sugar-72.gtkrc < $(EMPY)
+	$(PYTHON) -m em - -D scaling=\'72\' $(srcdir)/gtkrc.em > \
+		$(top_builddir)/data/sugar-72.gtkrc
 
 sugar-100.gtkrc: gtkrc.em
-	$(PYTHON) - -D scaling=\'100\' $(srcdir)/gtkrc.em > \
-		$(top_builddir)/data/sugar-100.gtkrc < $(EMPY)
+	$(PYTHON) -m em - -D scaling=\'100\' $(srcdir)/gtkrc.em > \
+		$(top_builddir)/data/sugar-100.gtkrc
 
 sugardir = $(pkgdatadir)/data
 sugar_DATA =			\

